_pages/en_US/installing-boot9strap-(usm).txt

```diff
@@ -66,9 +49,9 @@ In this section, you will trigger the BannerBomb3 exploit using the DSiWare Mana
 1. Power on your device
 1. Launch System Settings on your device
 1. Navigate to `Data Management` -> `DSiWare`-> `SD Card` ([image](/images/screenshots/bb3/dsiware-management.png))
-  + Your device should show the [Bannerbomb3 menu](/images/screenshots/bb3/usm-menu.png)
-  + If this step causes your device to crash, [follow this troubleshooting guide](troubleshooting#dsiware-management-menu-crashes-without-showing-usm-menu)
-1. Press (A) to select "Inject haxx"
+  + Your device should show the BB3 multihax menu
+  + If this step causes your device to crash, [follow this troubleshooting guide](troubleshooting#dsiware-management-menu-crashes-without-showing-bb3-multihax-menu-usm)
+1. Use the D-Pad to navigate and press the (A) button to select "Install unSAFE_MODE"
   + Your device will automatically power off
 
 #### Section III - unSAFE_MODE
```

Comment:
it won't let me specifically comment on the line but the safe mode inline troubleshooting should no longer be necessary. If you want to keep it you have to switch out the link from bb3 to fredtool

Reply:
Resolved.